README.md

```diff
@@ -138,7 +138,8 @@ $ roslaunch ca_driver create_1.launch [publish_tf:=true]
 `loop_hz`   |  Frequency of internal update loop |  `10.0`
 `dev`       |  Serial port |  `/dev/ttyUSB0`
 `publish_tf`|  Publish the transform between `odom` and `base_footprint` frames | `true`  
-`create_1`  |  Is robot model Create 1 (Roomba 400 series)?  | `false`
+`create_1`  |  Is robot model Create 1 (protocol version 2)?. Overrides `robot_model` if `true`  | `false`
+`robot_model` |  The name of the robot being controlled | `CREATE_2`
```

Comment:
Please document the three possible options: `roomba_400`, `create_1` and `create_2`  

Also, I think we may as well remove the `create_1` parameter altogether.